### PR TITLE
Block editor intuitive trash

### PR DESCRIPTION
**Pull request by matthewr6**

### CODE REVIEW

**Comment by iamunknown2** [comment]
Sorry, it was getting really late where I live (11PM).

**Comment by iamunknown2** [comment]
@Firedrake969 Since you don't like the whitespace getting into the way of the diff, here's a protip:

Add ?w=1 to the end of the URL (before the hash if there is any) to ignore the whitespace changes. For example:

https://github.com/ElementalCode/elemental/pull/213/files?w=1

**Comment by iamunknown2** [comment]
@Firedrake969 I have.

**Comment by matthewr6** [comment]
@ElementalCode/programmers someone else please test and merge I've been busy
(or @iamunknown2 make a gif of you using it)

**Comment by iamunknown2** [comment]
Here's the gif of me using it. This took quite a while, as I experimented with different settings.
intuitive_trash_demo

**Comment by matthewr6** [comment]
Could you replace the trash can in the lower right corner as well?  I think the plan was to just add that feature, not remove the other trash can (check this @liam4 )

**Comment by towerofnix** [comment]
+1, yeah, that was the plan.

**Comment by iamunknown2** [comment]
OK. Working on it.

**Comment by TheInitializer** [comment]
idea: while dragging block, put text like "Delete block" next to the garbage can in the bottom right

File: circle.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 deployment:
-  production:
-    branch: master
-    commands:
-        - git push git@heroku.com:elementalcode.git $CIRCLE_SHA1:master
-        - heroku run python manage.py migrate --app elementalcode
\ No newline at end of file
+    production:
+        branch: master
+	commands:
```
Comment (matthewr6) [inline circle.yml]: Don't use tabs, use spaces.

**Comment by iamunknown2** [comment]
@liam4 @Firedrake969 I can't get the trash in the bottom-left to show, although it seems to register in the browser (inspect element). I've looked at the z-indices, and it seems that the z-index of the trash can is the highest.

Any help?

**Comment by matthewr6** [comment]
Hm.  How hard was it to implement that palette trash can alone?  (copy/pastable easily?)

**Comment by iamunknown2** [comment]
@Firedrake969 Screenshot of the problem:
trashcan2_not_showing

**Comment by iamunknown2** [comment]
Any help?

**Comment by iamunknown2** [comment]
To your question @Firedrake969 I had to do some tweaking with blocks.js, style.scss and of course, HTML.

**Comment by iamunknown2** [comment]
I found out that I had forgot to put in

``` css
background-size: 100% auto;
```

:package: 
After changing it (and running `sass --watch`), it worked! Yay!

**Comment by iamunknown2** [comment]
Uhh what happened https://github.com/ElementalCode/elemental/pull/213/commits/d7463313477b0557b03b24bbc6f7d92714c10bbe I'm pretty sure I didn't change the HTML that much...

**Comment by iamunknown2** [comment]
Phew. Crisis over. :package:

**Comment by iamunknown2** [comment]
> idea: while dragging block, put text like "Delete block" next to the garbage can in the bottom right

Working on it.

**Comment by iamunknown2** [comment]
Apparently git isn't very good with detections in whitespace changes - it gave up and made an unaligned diff and made it impossible to do an automerge.

**Comment by matthewr6** [comment]
I'm super confused
Maybe just re-branch off gh-pages and re-implement the changes?

**Comment by iamunknown2** [comment]
@Firedrake969 I'm thinking of merging before changing the white-space of gh-pages.

**Comment by matthewr6** [comment]
I would suggest just re-branching and re-implementing, since it doesn't seem like implementing it was too hard.  (read as - do that instead!)

**Comment by iamunknown2** [comment]
@Firedrake969 I don't really mind about the whitespace. Rebranching won't help the whitespacing issue either.

The implementation was pretty difficult - I had to add and change a few functions in blocks.js, other than changing the SCSS (which was less difficult than the JS) and the HTML (which was the easiest).

**Comment by matthewr6** [comment]
Yeah it will - _keeping the whitespace the same as before_

So then what we (ie you probably) will do is copy/paste the raw blocks.js, scss files, and the html files from this branch into another branch from gh-pages so it doesn't become this mess of commits and reverted commits :P

**Comment by iamunknown2** [comment]
I'll work on it tomorrow.

**Pull request closed without merging**